torch/csrc/jit/fuser/executor.cpp

```diff
@@ -358,34 +370,191 @@ bool runFusion(const int64_t key, Stack& stack) {
   if (!maybe_map_size)
     return false;
   if (spec.hasRandom()) {
-      bool hasBroadcast = shouldExpandArgs(spec,inputs, *maybe_map_size);
-      if (hasBroadcast) return false;
+    bool hasBroadcast = shouldExpandArgs(spec, inputs, *maybe_map_size);
```

Comment:
Please please please please avoid formatting changes!

Reply:
Sorry, I have to get smarter about clang-format... :disappointed: